diff --git a/node_modules/lua-format/src/luamin.js b/node_modules/lua-format/src/luamin.js
index fcf42ff..a83263a 100644
--- a/node_modules/lua-format/src/luamin.js
+++ b/node_modules/lua-format/src/luamin.js
@@ -4198,7 +4198,7 @@ function BeautifyVariables(globalScope, rootScope, renameGlobals) {
 }
 
 
-let watermark = `--discord.gg/boronide, code generated using luamin.js™\n\n`
+//let watermark = `--discord.gg/boronide, code generated using luamin.js™\n\n`
 
 let luaminp = {}
 
@@ -4218,7 +4218,7 @@ luaminp.Minify = function(scr, options) {
     StripAst(ast)
 
     let result = PrintAst(ast)
-    result = `${watermark}\n\n${result}`
+    //result = `${watermark}\n\n${result}`
 
     return result
 }
@@ -4237,7 +4237,7 @@ luaminp.Beautify = function(scr, options) {
     FormatAst(ast)
 
     let result = PrintAst(ast)
-    result = `${watermark}\n\n${result}`
+    //result = `${watermark}\n\n${result}`
 
     return result
 }
