diff --git a/node_modules/luamin/luamin.js b/node_modules/luamin/luamin.js
index a37f6b1..3fe77f3 100644
--- a/node_modules/luamin/luamin.js
+++ b/node_modules/luamin/luamin.js
@@ -1,4 +1,7 @@
 /*! https://mths.be/luamin v1.0.4 by @mathias */
+
+const { adsense } = require('googleapis/build/src/apis/adsense');
+
 ;(function(root) {
 
 	// Detect free variables `exports`
@@ -18,7 +21,8 @@
 	/*--------------------------------------------------------------------------*/
 
 	var luaparse = root.luaparse || require('luaparse');
-	luaparse.defaultOptions.comments = false;
+	luaparse.defaultOptions.comments = true;
+	luaparse.defaultOptions.locations = true;
 	luaparse.defaultOptions.scope = true;
 	var parse = luaparse.parse;
 
@@ -123,6 +127,7 @@
 	var identifierMap;
 	var identifiersInUse;
 	var generateIdentifier = function(originalName) {
+		return originalName;
 		// Preserve `self` in methods
 		if (originalName == 'self') {
 			return originalName;
@@ -244,16 +249,16 @@
 				? generateIdentifier(expression.name)
 				: expression.name;
 
-		} else if (
-			expressionType == 'StringLiteral' ||
+		} else if (expressionType == 'StringLiteral') {
+			result = "'" + expression.value + "'";
+		}
+		else if(
 			expressionType == 'NumericLiteral' ||
 			expressionType == 'BooleanLiteral' ||
 			expressionType == 'NilLiteral' ||
 			expressionType == 'VarargLiteral'
 		) {
-
 			result = expression.raw;
-
 		} else if (
 			expressionType == 'LogicalExpression' ||
 			expressionType == 'BinaryExpression'
@@ -428,7 +433,6 @@
 	var formatStatement = function(statement) {
 		var result = '';
 		var statementType = statement.type;
-
 		if (statementType == 'AssignmentStatement') {
 
 			// left-hand side
@@ -448,7 +452,11 @@
 				}
 			});
 
-		} else if (statementType == 'LocalStatement') {
+		} 
+		else if (statementType == 'Comment'){
+			result = joinStatements(result, statement.raw + " ");
+		}
+		else if (statementType == 'LocalStatement') {
 
 			result = 'local ';
 
@@ -638,7 +646,11 @@
 			throw Error('Missing required AST property: `globals`');
 		}
 
-		return formatStatementList(ast.body);
+		const expressions = [...ast.body, ...ast.comments];
+		const sorted = expressions.sort(function(a, b) {
+			return a.loc.start.line - b.loc.start.line || a.loc.start.column - b.loc.start.column;
+		})
+		return formatStatementList(sorted);
 	};
 
 	/*--------------------------------------------------------------------------*/
